### PR TITLE
Add female name strings for drake units

### PR DESCRIPTION
**Pull request by Jonathan-Kelly**
This (should) solve the issue raised by Antro of the Italiano translation team [on the forums](https://forums.wesnoth.org/viewtopic.php?p=684505#p684505). Basically, some languages have no neuter gender or cannot use it for unit names without sounding very awkward. So if any drake character is female, then its `unit_type` must also have a translatable string available for the female version of its name.

As with the naga race, there is no need for new sprites to be created for female variations of drake units. Since our real life audience is entirely made up of humans, they will not be able to tell the difference between male and female dragons anyway. :p

Additionally, this enables the long unused female drake name list and name generator to be used for recruited and randomly spawned units.

### CODE REVIEW

File: data/core/units/drakes/Arbiter.cfg [hunk 1]
```diff
@@ -169,4 +170,8 @@
             image="units/drakes/arbiter.png:100"
         [/frame]
     [/attack_anim]
+    [female]
+        name= _ "Drake Arbiter"
```
Comment (Pentarctagon) [inline data/core/units/drakes/Arbiter.cfg:174]: These all be `female^Drake XYZ`, otherwise they won't appear as separate strings for translation.

Comment (Jonathan-Kelly) [reply data/core/units/drakes/Arbiter.cfg]: Good catch, thanks! I used the naga fighter as a starting point but that actually replaces the -a suffix with a (Sanskrit?) -ini for its female variant.

**Comment by CelticMinstrel** [comment]
> Since our real life audience is entirely made up of humans, they will not be able to tell the difference between male and female dragons anyway.

You missed the adjective in front of "humans". Obviously it's possible for a human to distinguish genders in radically different species if they've trained in that field. ;p

> Additionally, this enables the long unused female drake name list and name generator to be used for recruited and randomly spawned units.

Please reword the commit message for this (the 2nd commit). The current message doesn't get across the concept that you can now recruit female drakes. It's not about enabling "names".

I'll also ask, is the plan to make drakes completely gender-indifferent? That is, all units are allowed to recruit female variations? I'm completely fine with that, just want to make sure it was done on purpose and not just absent-mindedly.

**Comment by Pentarctagon** [comment]
@Jonathan-Kelly btw just to confirm - you're sigurdfd on Discord and SigurdFireDragon on the forums, correct?

**Comment by CelticMinstrel** [comment]
That's incorrect. He is "name" on the forums and "deus" on Discord.

**Comment by Pentarctagon** [comment]
Ah, alright, that makes more sense. Not sure what I was thinking.

**Comment by Jonathan-Kelly** [comment]
> Please reword the commit message for this (the 2nd commit). The current message doesn't get across the concept that you can now recruit female drakes. It's not about enabling "names".

Alright, reworded it.

> I'll also ask, is the plan to make drakes completely gender-indifferent? That is, all units are allowed to recruit female variations? I'm completely fine with that, just want to make sure it was done on purpose and not just absent-mindedly.

Yeah, it is deliberate. I think it is interesting for each major faction to have significant differences from the others. This is just one of the things that helps distinguish the lawful drakes from the lawful humans (loyalists) in terms of culture. And it fits the idea of drakes being obsessed with meritocracy (as part of their wider blue-and-orange morality).

**Comment by demario-w** [comment]
> And it fits the idea of drakes being obsessed with meritocracy (as part of **their wider blue-and-orange morality**).

[Let's remember](https://github.com/wesnoth/wesnoth/issues/7361) how you escaped accountability for the most despicable aspects for the drake lore defined in WoF by saying they were [very specific](https://github.com/wesnoth/wesnoth/issues/7361#issuecomment-1430146149) to that time and space, and did not apply to the drake race in general.

It seems you are now willing to generalize your "creation" to the full race. Not that it was [not expected](https://github.com/wesnoth/wesnoth/issues/7361#issuecomment-1430610448).

**Comment by Pentarctagon** [comment]
Is this ready to merge then? I'm waiting on this and #8056/#8031 before tagging 1.17.23.

**Comment by Jonathan-Kelly** [comment]
I think it should be good to merge.

**Pull request merged**